core/src/main/scala/ProtoCodeGen.scala

```diff
@@ -61,7 +61,7 @@ object ProtoCodeGen {
           ProtoAnnotationsProcessor[Try]
             .process(inputFile)
             .map {
-              case pd if pd.options.nonEmpty && pd.messages.nonEmpty && pd.services.nonEmpty =>
+              case pd if pd.options.nonEmpty || pd.messages.nonEmpty || pd.services.nonEmpty =>
```

Comment:
I'm still learning about protocol buffers, but does a Scala "model" need all 3 types in one file?

Reply:
Good catch 👍